collections/_posts/2022-08-31-MAE.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: review
+title: "Masked Autoencoders Are Scalable Vision Learners"
+tags: deep-learning attention transformer self-supervised
+author: "Gaspard Dussert"
+cite:
+    authors: "Kaiming He, Xinlei Chen, Saining Xie, Yanghao Li, Piotr Dollár, Ross Girshick"
+    title:   "Masked Autoencoders Are Scalable Vision Learners"
+    venue:   "IEEE/CVF Conference on Computer Vision and Pattern Recognition (CVPR), 2022, pp. 16000-16009"
+pdf: "https://openaccess.thecvf.com/content/CVPR2022/papers/He_Masked_Autoencoders_Are_Scalable_Vision_Learners_CVPR_2022_paper.pdf"
+---
+
+# Highlight
+
+* Masked Auto-Encoder is a very simple self-supervised training approach giving state-of-the-art results
```

Comment:
I would change it to "Masked Auto-Encoder is a very simple self-supervised training approach that gives state-of-the-art results"